Improve the instructions in the readme.md file.

barryd15 opened the pull request:
Mainly trying to address problems and challenges found by the early builders.

barryd15 committed:
Improve the instructions in the readme.md file.

Mainly trying to address problems and challenges found by the early builders.

barryd15 commented:
Please review my noob soldering tips (eg. regarding paper towels).  Might need further editing in that section.  But we got the important details about how to orient the components where it's not obvious.  That's what's most important.
Also the warning about checking for missing components due to insufficient stock.  Very important, that.

michd commented on readme.md:
```diff
@@ -36,7 +44,13 @@ The Panel Boards purpose is to implement three subsystems underneath a standard
 3. Order the remaining through-hole parts - A list of these parts can be seen at `release/unpopulated.csv`. Companies that sell parts like this include Mouser, Digikey, RS-Components and Farnell.
 
-4. Solder the remaining parts onto the board. Fairly evident where each of them go, though a more detailed guide here could be helpful. Pull requests welcome. :-)
+4. Solder the remaining parts onto the board.  Here is a guide for each through-hole part on the panel board:
+    J1 / Würth Elektronik 694106301002 / 2.1mm DC barrel jack connector: Easy.  Big contacts can take a lot of solder.  I put a folded paper towel under it to keep it pressed against the board while soldering, even when other components protrude a little further from the board.
+    J2 / Amphenol FCI 54602-908LF / Rj45/ethernet connector: Snaps onto the board with plastic tabs so you don't have to worry about holding it in place.  These large components can make soldering other components more difficult, so it may be a good idea to leave the big ones until last.
+    J3 / 3M 30310-6002HB / 10-pin ST-LINK connector with plastic fence: The gap in the fence goes on the side with pin 1 - the pin with the square solder pad.  The gap also faces towards the edge of the board.  I used a folded paper towel under this one.
+    J4 / Harwin M20-9980446 / External Logic connector: It's quite possible you'll never use this connector.  Use a folded paper towel to help hold it in place.  Or leave it off the board entirely, which will probably lower the chances of damaging the 74HCT245 chip next to it with a static discharge.
+    J5,J6,J7,J8 / Japan Solderless Terminals B4B-PH-K-S(LF)(SN) / load cell sensor connectors: The gap in the fence goes towards the inside of the board.  If you're looking at the side of the connector with that gap, pin 1 is on the right.  Unless you're holding it upside-down.  Just point the gap towards the middle of the board and you'll be okay.These things grip to the board quite well.
+    SW1,SW2 / TE Connectivity 1825910-6: The pins lock into the board so you don't have to worry about it falling out of position.  When these switches are placed correctly, the pins come out on the left and right sides.
```
Bunch of welcome details here, thank you! I like to use blu-tack to hold components in place instead of padding with paper towel, although, trying not to let the blu-tack touch the pins-- it gets a little goopy  when heated, harder to remove. Another common alternative to hold components in place is to tape them down with heat-resistant tape like kapton tape.

As for suggestions, maybe you could add in those alternatives too when first mentioning the paper towel method? (I'm also a little wary of paper near something as high-temperature as a soldering iron, possibly a fire hazard)

michd reviewed (commented):
Thanks! Looks great, just left one suggestion for adding alternative ways to keep through-hole components in place for soldering.

barryd15 committed:
Change the presentation of questionable techniques to a style of "here's an idea, but I'm no expert and you're ultimately the one responsible for your soldering".

Important because a reader could easily make the mistake of assuming a single author who is experienced in soldering.

barryd15 commented:
I adjusted the presentation a bit so there's no risk we accidentally present questionable methods as if from an expert.  Included your suggestions, but basically scaled that whole bit back so they're all just presented as ideas, leaving any judgement up to the reader.  Of course the topic could be expanded again in future contributions by myself or others.

Still new to github collaboration by the way.  Will probably make an explicit feature branch for future contributions.

michd commented:
Thanks for these updates Barry, looks great! 

And yeah, if you hang on to your fork of the repository it will be easier to make new feature branches in future.
